wp_api/src/ssl.rs

```diff
@@ -81,3 +89,9 @@ pub struct SSLCertificateIssuer {
     pub organization: Option<String>,
     pub country: Option<String>,
 }
+
+impl From<x509_cert::time::Time> for WpGmtDateTime {
+    fn from(date_time: x509_cert::time::Time) -> Self {
+        WpGmtDateTime::from_timestamp(date_time.to_unix_duration().as_secs() as i64)
```

Comment:
The original PR had the following implementation:
```
impl From<u64> for WpGmtDateTime {
    fn from(seconds: u64) -> Self {
        Self::from_timestamp(seconds.try_into().unwrap())
    }
}
```

I like simply casting it to `i64` here because it's a clean cast. The alternative is to add a `From` implementation, but I don't think this ^ `From` implementation is the best way to do it, we should instead cast to `i64` and forward to the existing implementation.

Reply:
u64 -> i64 should always have a loss of precision, no?

Reply:
Technically yes, practically no because this is a date. Regardless, there is no way to avoid that because the function we use [chrono::DateTime::from_timestamp](https://docs.rs/chrono/latest/chrono/struct.DateTime.html#method.from_timestamp) accepts `i64`.

This is actually why I preferred having the cast here as we know that `date_time` is a valid date and can be cast without a problem. If we add a `From<u64>` implementation then it's less clear how it should be handled.